docs/docs/client-guide/defining-scheduler-version.md

```diff
@@ -0,0 +1,2 @@
+# Defining Scheduler Version
+Optimus provide capability to determine the airflow version per project by defining `scheduler_version` in project config (`optimus.yaml`). By default, optimus use airflow version 2.1.4 if not specifying in `optimus.yaml` config.
```

Comment:
`if it is not specified`

Reply:
is this supports all Airflow versions or only > Airflow 2.0?

Reply:
got it. for now, it only support airflow `2.1` and `2.4`